apps/validator/src/main/scala/org/lfdecentralizedtrust/splice/validator/domain/DomainConnector.scala

```diff
@@ -192,8 +192,20 @@ class DomainConnector(
           // so this is just an extra safeguard.
           sequencers.synchronizerId == decentralizedSynchronizerId
         )
+      val sv_filteredSequencers = config.domains.global.sequencerNames match {
```

Comment:
lets make sure we follow scala naming conventions

```suggestion
      val svFilteredSequencers = config.domains.global.sequencerNames match {
```

Reply:
Done